controllers/machinedeletion_controller.go

```diff
@@ -63,19 +73,54 @@ func (r *MachineDeletionReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	if remediation = r.getRemediation(ctx, req); remediation == nil {
 		return ctrl.Result{}, nil
```

Comment:
I'd expect a log message if there was a problem fetching the cr

Reply:
I'm already logging inside the method for use cases of `!errors.IsNotFound(err)`